tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -137,6 +137,11 @@ sealed trait NativeConfig {
     }
   }
 
+  /** Shall enable the use of C++ runtime. Useful if you want to compile c++
+   *  code in your Scala Native application.
+   */
+  def useCPPRuntime: Boolean
```

Comment:
This is actually a lie because windows can't function without the cpp runtime at the moment, so should I rename this? Do you have suggestions?

Reply:
Default to true on Windows?

Reply:
The current code defaults to `false` and communicates the "intention" to use the c++ runtime. But on Config.scala this is added with `|| config.targetsWindows`.
But I accept suggestions on how to improve the config.